Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM node:5.8.0
+FROM node:8.1.0
```

Comment:
Now that we're on 8.1.0 which is supported by alpine linux, do you mind trying to use that image instead? If it's too much out of scope then don't worry about it.